docs/weightonlyquant.md

```diff
@@ -186,12 +187,12 @@ tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
 prompt = "Once upon a time, there existed a little girl,"
 inputs = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
 
-qmodel = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True, device_map="xpu", trust_remote_code=True)
+qmodel = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True, device_map="xpu", trust_remote_code=True, use_llm_runtime=False)

```

Comment:
shall we make it more smart, e.g., for IPEX supported optimized model, it goes to IPEX; otherwise, it goes to ITREX OOB mode. Don't expose this parameter to users for GPU case. @a32543254 @kevinintel

Reply:
Zhenwei,
we can create a model list for ipex.
Currently, only Qwen

Reply:
At present, we cannot do it because the optimized model of IPEX cannot call the `save` interface. We need to wait for IPEX to support the `save` interface